packages/adblocker/src/engine/bucket/network.ts

```diff
@@ -28,12 +28,20 @@ export default class NetworkFilterBucket {
       config,
     );
 
+    bucket.htmlIndex = ReverseIndex.deserialize(
```

Comment:
We should also not forget to increase the ENGINE_VERSION.

Reply:
ENGINE_VERSION auto updates by the CI

Reply:
Does it also get updated without running a codebook compression update?